checkout/checkout_service.bal

```diff
@@ -15,10 +15,11 @@
 // under the License.
 
 import ballerina/grpc;
-import ballerina/uuid;
 import ballerina/log;
+import ballerina/uuid;
 import ballerinax/jaeger as _;
 import wso2/client_stubs as stub;
+import wso2/money_utils as utils;
```

Comment:
How about using `money` as the prefix instead?

Reply:
Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/65/commits/e746edee6815377481f762a558e106b9e16d715b